### DISCUSSION TITLE
JavaScript not loading after livewire defer loading

### DISCUSSION THREAD

**Discussion by notbotman** [livewire/livewire#2318] [1. Help]
JavaScript not loading after livewire defer loading

Here I used Tippy js library

The page should show popup on hovering over the user, but is it not showing after defer loading the components of the page.

![image](https://user-images.githubusercontent.com/74244707/103904347-2a480400-5123-11eb-9b8c-cb46cb7e63cc.png)

I think I'm missing some event listeners/livewire js hooks!

Any help how can I fix this!

**Comment by joshhanley** [comment]
@notbotman can you also share your relevant blade code of your user section you are trying to hover over?

The problem is likely to be morphdom deleting the tippy generated code. The reason is if the code wasn't generated by Livewire, it doesn't know about it, so it gets removed next time an update happens. That's what wire:ignore does, that insures anything inside an element which has wire:ignore on it doesn't get changed, so libraries like this can insert DOM as it needs without it being deleted.

Hope this helps!

**Reply by notbotman** [reply]
Hi @joshhanley,

Here is the code, Tippy is triggered using this `user-popover` class, I tried adding `wire:ignore` but didn't work 😞 

Here is the website http://taskord.com where the problem occurs

If you opened the page that does not defer load the tippy works fine Eg: https://taskord.com/task/1237

```
<a
    href="{{ route('user.done', ['username' => $task->user->username]) }}"
    class="fw-bold text-dark user-popover"
    data-id="{{ $task->user->id }}"
>
    @if ($task->user->firstname or $task->user->lastname)
        {{ $task->user->firstname }}{{ ' '.$task->user->lastname }}
    @else
        {{ $task->user->username }}
    @endif
    @if ($task->user->isVerified)
        <x-heroicon-s-badge-check class="heroicon ms-1 text-primary verified" />
    @endif
    @if ($task->user->isPatron)
        <a class="patron" href="{{ route('patron.home') }}" aria-label="Patron">
            <x-heroicon-s-star class="heroicon text-gold" />
        </a>
    @endif
    <div class="small text-secondary fw-normal">{{ "@" . $task->user->username }}</div>
</a>
```

**Reply by joshhanley** [reply]
@notbotman sorry I missed your reply above! Did you manage to get it working?
